docs/user/explanations/advanced.md

```diff
@@ -106,6 +74,134 @@ installing any of the Linkerd control plane components:
+  API server from the previous point is one of them.
+
+Epinio cli functionality is implemented using the endpoints provided by the Epinio API server
+component. For example, when the user asks Epinio to "push" and application, the
+cli will contact the "Upload", "Stage" and "Deploy" endpoints of the Epinio API to upload the application code,
+create a container image for the application using this code and run the application on the cluster.
+
+The Epinio API server is running on the cluster and made accessible using Kubernetes resources like
+Deployments, Services,  Ingresses and Secrets.
+
+### Cert Manager
+
+[Upstream documentation](https://cert-manager.io/docs/)
+
+The Cert manager component is deployed by Epinio and use to generate and renew the various Certificates needed in order to
+serve the various accessible endpoints over TLS (e.g. the Epinio API server).
+
+Epinio supports various options when it comes to certificate issuers (let's encrypt, private CA, brings your own CA, self signed certs).
+Cert Manager simplifies the way we handle the various different certificate issuers within Epinio.
+
+You can read more about certificate issuers here: [certificate issuers documentation](../howtos/certificate_issuers.md)
+
+### Kubed
+
+[Upstream documentation](https://github.com/kubeops/kubed)
+
+Kubed is installed used by Epinio, in order to keep secrets that are needed in more than
+one namespace, synced. For example, the TLS certificate of the [Container Registry](#container-registry) is also needed
+in the Tekton staging namespace, in order to trust the certificate when pushing the application images.
+
+Kubed makes sure that if the source secret changes, the copy will change too.
+
+Warning: this doesn't mean things will still work if you re-generate a secret manually. Secret rotation will be handled by Epinio in the future.
+
+### Google Service Broker
+
+[Upstream project link](https://github.com/GoogleCloudPlatform/gcp-service-broker)
+
+Many applications need additional services in order to work. Those services could be databases, cache servers, email gateways, log collecting applications or other.
+The various cloud providers already offer such services and being able to use them in the various Platforms as a Service (PaaS) in a unified way,
+gave birth to the concept of the (Open Service Broker API - aka osbapi)[https://www.openservicebrokerapi.org/]. This concept is adopted by various
+providers among which, Google. There seems to be a shift towards the "Operator" pattern, e.g. Google seems to have moved its focus to the [Config Connector](https://cloud.google.com/config-connector/docs/overview) as described at the top of the README [here](https://github.com/googlearchive/k8s-service-catalog).
+
+Since Google service broker still works and since the common interface provided the the [Service Catalog](#service-catalog) makes it easy to integrate, Epinio
```

Comment:
```suggestion
Since Google's service broker still works and since the common interface provided by the [Service Catalog](#service-catalog) makes it easy to integrate, Epinio
```